crates/compiler/test_gen/src/helpers/dev.rs

```diff
@@ -192,7 +192,7 @@ pub fn helper(
         .expect("failed to build output object");
     std::fs::write(&app_o_file, module_out).expect("failed to write object to file");
 
-    // std::fs::copy(&app_o_file, "/tmp/app.o").unwrap();
+    std::fs::copy(&app_o_file, "/tmp/app.o").unwrap();
```

Comment:
Do we have a debug flag or something of that nature we could put this behind since it is so common to enable when working on the dev backend?